charts/apisix-dashboard/templates/ingress.yaml

```diff
@@ -24,6 +24,9 @@ apiVersion: extensions/v1beta1
 {{- end }}
 kind: Ingress
 metadata:
+  {{- if .Values.ingress.classNmae }}
+  ingressClassName: {{- .Values.ingress.classNmae -}}
```

Comment:
There is a minimum version constraint for Kubernetes, Please update the document to classify it.

Reply:
values.yaml  file describe "ingressClassName"  kubernetes version ?

Reply:
Yes, but I think readme should also be updated.

Reply:
hi,readme.md already update;